File: ch.jnetwork.sac-backend/Api/Model/Browser/BrowserNodeNewRequest.cs
namespace AngularPrototype.Api.Model.Browser
{
    /// <summary>
    /// Request zum anlegen eines neuen Ordners
    /// </summary>
    public class BrowserNodeNewRequest
    {
        /// <summary>
        /// Pfad in welchem der Ordner erstellt werden soll
        /// </summary>
        public string Path { get; set; }
        /// <summary>
        /// Neuer Name des Ordners
        /// </summary>
        public string NewFoldername { get; set; }
    }
}

File: ch.jnetwork.sac-controls/projects/sac-common/src/components/browser/models/browsernodenewrequest.ts
/**
 * Klasse für API Request zum anlegen eines neuen Nodes
 */
export class BrowserNodeNewRequest {
    // #region Properties

    /**
     * Name des neuen Nodes
     */
    public NewFoldername: string;

    /**
     * Pfad
     */
    public Path: string;

    // #endregion Properties

    // #region Constructors

    /**
     * Konstruktor
     * @param init Initiale Werte
     */
    constructor(init?: Partial<BrowserNodeNewRequest>) {
        Object.assign(this, init);
    }

    // #endregion Constructors
}
